ci-configs/packages-latest.json

```diff
@@ -563,6 +563,9 @@
     {
       "name": "@azure/communication-sms"
     },
+    {
```

Comment:
This is a preview package yes? It's already been added [over here](https://github.com/MicrosoftDocs/azure-docs-sdk-node/blob/9fa7add4f28c553149f2c6535194a62dc912a042/ci-configs/packages-preview.json#L133).

Reply:
Yes. I didn't know there's another file for preview packages.

Reply:
Fixed!